cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -536,6 +549,15 @@ func main() {
 		RefreshCfgThrottle: refreshCfgThrottle,
 	}
 
+	for _, instanceConfig := range cfgs {
```

Comment:
We should definitely flag this behavior by default (I think defaulting to disabled, at least for now).